<ui version="4.0" >
 <class>RefUi</class>
 <widget class="QDialog" name="RefUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="findKeysLA" >
       <property name="text" >
        <string>Fil&amp;ter:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>findLE</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findLE" >
       <property name="toolTip" >
        <string>Enter string to filter the label list</string>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="csFindCB" >
       <property name="toolTip" >
        <string>Filter case-sensitively</string>
       </property>
       <property name="text" >
        <string>Case-sensiti&amp;ve</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="updatePB" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Update the label list</string>
       </property>
       <property name="text" >
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="sortCB" >
         <property name="toolTip" >
          <string>Sort labels in alphabetical order (case-insensitively unless the Case-sensitive option is checked)</string>
         </property>
         <property name="text" >
          <string>&amp;Sort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="caseSensitiveCB" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Sort labels case-sensitively in alphabetical order</string>
         </property>
         <property name="text" >
          <string>Cas&amp;e-sensitive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="groupCB" >
         <property name="toolTip" >
          <string>Group labels by prefix (e.g. "sec:")</string>
         </property>
         <property name="text" >
          <string>Grou&amp;p</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="gotoPB" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Go to Label</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="refsL" >
     <property name="text" >
      <string>La&amp;bels in:</string>
     </property>
     <property name="buddy" >
      <cstring>bufferCO</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QComboBox" name="bufferCO" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="nameL" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>nameED</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2" >
    <widget class="QLineEdit" name="nameED" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="typeLA" >
     <property name="text" >
      <string>&amp;Format:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>typeLA</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2" >
    <widget class="QComboBox" name="typeCO" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Cross-reference as it appears in output</string>
     </property>
     <item>
      <property name="text" >
       <string>&lt;reference></string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>(&lt;reference>)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>&lt;page></string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>on page &lt;page></string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>&lt;reference> on page &lt;page></string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Formatted reference</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Textual reference</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="referenceL" >
     <property name="text" >
      <string>&amp;Label:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>referenceED</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QLineEdit" name="referenceED" />
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTreeWidget" name="refsTW" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bufferCO</tabstop>
  <tabstop>sortCB</tabstop>
  <tabstop>updatePB</tabstop>
  <tabstop>gotoPB</tabstop>
  <tabstop>referenceED</tabstop>
  <tabstop>typeCO</tabstop>
  <tabstop>nameED</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>applyPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
